PhysicsTools/PatAlgos/plugins/PATMuonProducer.h

```diff
@@ -34,6 +34,10 @@
 #include "TrackingTools/TransientTrack/interface/TransientTrackBuilder.h"
 #include "PhysicsTools/PatAlgos/interface/MuonMvaEstimator.h"
 #include "PhysicsTools/PatAlgos/interface/SoftMuonMvaEstimator.h"
+#include "DataFormats/PatCandidates/interface/TriggerObjectStandAlone.h"
+#include "Geometry/Records/interface/GlobalTrackingGeometryRecord.h"
+// #include "DataFormats/GeometryVector/interface/GlobalVector.h"
+// #include "DataFormats/GeometryVector/interface/LocalVector.h"
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant